### PR TITLE
Merge CHANGELOG files into one, update RELEASING instructions to include making GitHub release.

### PR DESCRIPTION
**Pull request by zach-klippenstein**
When you create a release on GitHub, you can provide a rich markdown description of the release that will show up directly on the GitHub releases page. I have done this for a single release, and it looks much better. I've updated the RELEASING instructions to include this step. This also requires merging the kotlin and swift changelogs (since the GitHub interface only gives you one location), so we might as well merge our CHANGELOG files as well.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,220 @@
...
+ * Upgrade Gradle to 5.x ([#113](https://github.com/square/workflow/issues/113))
+ * Upgrade AGP to 3.3.0 ([#115](https://github.com/square/workflow/issues/115))
+
+## Version 0.12.0
+
+_2019-4-10_
+
+# Swift
+
+ * Improve type safety by removing AnyScreenViewController as a base type. ([#200](https://github.com/square/workflow/pull/200))
```
Comment (rjrjr) [inline CHANGELOG.md]: Do we still need to include the issue URLs? Hoping that the pretty rendered release pages will do that for us and we can delete them.

**Comment by zach-klippenstein** [comment]
Old: 
image

New:
image

File: CHANGELOG.md (hunk 1 above)
Comment (zach-klippenstein) [reply CHANGELOG.md]: We don't need them in the github description. They don't link from regular changelog markdown but that's probably fine, nobody's gonna use the links in this file anyway.

**Comment by zach-klippenstein** [comment]
Uhoh looks like we've got a flaky test. Filed #302.

**Pull request merged**